src/org/opendatakit/briefcase/util/WebUtils.java

```diff
@@ -307,39 +315,37 @@ public static final HttpPost createOpenRosaHttpPost(URI uri) {
 	}
 
 	public static final HttpClient createHttpClient() {
-	  // configure connection
-	  SocketConfig socketConfig = SocketConfig.copy(SocketConfig.DEFAULT).setSoTimeout(SERVER_CONNECTION_TIMEOUT).build();
-	  
-     // if possible, bias toward digest auth (may not be in 4.0 beta 2)
-     List<String> targetPreferredAuthSchemes = new ArrayList<String>();
-     targetPreferredAuthSchemes.add(AuthSchemes.DIGEST);
-     targetPreferredAuthSchemes.add(AuthSchemes.BASIC);
-
-     RequestConfig requestConfig = RequestConfig.copy(RequestConfig.DEFAULT)
-	      .setConnectTimeout(SERVER_CONNECTION_TIMEOUT)
-	      // support authenticating
-	      .setAuthenticationEnabled(true)
-	      // support redirecting to handle http: => https: transition
-	      .setRedirectsEnabled(true)
-	      .setMaxRedirects(1)
-	      .setCircularRedirectsAllowed(true)
-	      .setTargetPreferredAuthSchemes(targetPreferredAuthSchemes)
-	      .build();
-	
-     CloseableHttpClient httpClient;
-     HttpHost proxy = BriefcasePreferences.getBriefCaseProxyConnection();
-     if (proxy == null) {
-     httpClient = HttpClientBuilder.create()
-             .setDefaultSocketConfig(socketConfig)
-             .setDefaultRequestConfig(requestConfig).build();
-     } else {
-         DefaultProxyRoutePlanner routePlanner = new DefaultProxyRoutePlanner(proxy);
-     httpClient = HttpClientBuilder.create()
-             .setDefaultSocketConfig(socketConfig)
-             .setDefaultRequestConfig(requestConfig)
-             .setRoutePlanner(routePlanner).build();
-     }
-      return httpClient;
+		// configure connection
+		SocketConfig socketConfig = SocketConfig.copy(SocketConfig.DEFAULT).setSoTimeout(SERVER_CONNECTION_TIMEOUT).build();
+
+		// if possible, bias toward digest auth (may not be in 4.0 beta 2)
+		List<String> targetPreferredAuthSchemes = new ArrayList<String>();
+		targetPreferredAuthSchemes.add(AuthSchemes.DIGEST);
+		targetPreferredAuthSchemes.add(AuthSchemes.BASIC);
+
+		RequestConfig requestConfig = RequestConfig.copy(RequestConfig.DEFAULT)
+				.setConnectTimeout(SERVER_CONNECTION_TIMEOUT)
+				// support authenticating
+				.setAuthenticationEnabled(true)
+				// support redirecting to handle http: => https: transition
+				.setRedirectsEnabled(true)
+				.setMaxRedirects(1)
+				.setCircularRedirectsAllowed(true)
+				.setTargetPreferredAuthSchemes(targetPreferredAuthSchemes)
+				.build();
+
+		HttpClientBuilder clientBuilder = HttpClientBuilder.create()
+				.setConnectionManager(connectionManager)
+				.setConnectionManagerShared(true)
+				.setDefaultSocketConfig(socketConfig)
+				.setDefaultRequestConfig(requestConfig);
+
+		HttpHost proxy = BriefcasePreferences.getBriefCaseProxyConnection();
+		if (proxy != null) {
+			clientBuilder.setRoutePlanner(new DefaultProxyRoutePlanner(proxy)).build();
```

Comment:
Yes, you're right. At the very least, it's confusing. I just missed this when converting the original code. I'll remove it.